sled-agent/src/rack_setup/service.rs

```diff
@@ -1449,6 +1453,9 @@ pub(crate) fn build_initial_blueprint_from_sled_configs(
         cockroachdb_fingerprint: String::new(),
         cockroachdb_setting_preserve_downgrade:
             CockroachDbPreserveDowngrade::DoNotModify,
+        clickhouse_cluster_config: ClickhouseClusterConfig::new(
+            rack_id.to_string(),
```

Comment:
This is probably wrong. We probably want a distinct id in case clickhouse clusters can span racks in a multirack world.